### PR TITLE
Pranjaltale16 patch 2

### PR DESCRIPTION
**Pull request by pranjaltale16**

### CODE REVIEW

File: tale.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This is a glimpse into the life of the one true Lord, our very own _Sri Sri Parampujya Mahatma Pranjal Tale Ji_.
 
-## Covfefe
+## Covfefe or tea
```
Comment (rheaparekh) [inline tale.md:5]: @dhruvkb I think this is incorrect. Please double check and correct this.

Comment (dhruvkb) [reply tale.md:5]: We are mere followers. The Lord himself has blessed tea to the rank of one of the two promised beverages so all we can do is agree.

Comment (kkpatel1) [reply tale.md:5]: I hope the Lord can handle the Covfefe followers and Tea followers simultaneously. I am glad that God agrees with the Tea and hence, now completely absorbs me without any expectation of change!

Comment (dhruvkb) [reply tale.md:5]: @kartikp1995 @rheaparekh Our Lord is also considering to adopt Bournvita people into his religion.

**Pull request merged**